### PR TITLE
feature/new uikit/122

### LINKED ISSUE #122: Haberler ve İş Sayfası
**Description**
burada iki sayfa tablı bir yapıda içeride aynı yerde olacak

- sayfanın üst ortasında ikili tab olacak ve tıklandığında değişecek, scroll ile yan taba geçilemeyecek
- üst sayfadaki tab sticky olacak scrool edilidğinde üste çıkacak ama yukarıda daima sabit kalacak
- iki sayfa da tıklandığında shimmer yüklenme olacak container card olarak gözükecek
- ilk tab haberler olacak
- haberler cardı ekteki gibi olacak
- Burada haberin resmi title paylaşan kim ve ufak bir kabartma ile gözükecek 
image
- Burada haberi paylaşandan ziyade kaynak gibi göstericez

- İş kartı ekteki gibi olacak
- 
image

- bunun solunda resim sağda title ve altında ne zaman paylaşıldığı tarihi tutulacak 

**Screen Preview**
image

### PR DESCRIPTION
**Pull request by VB10**
- home page added
- implements general properties
- added favorite page
- added comment updates
- update with feedback
- update reminder notes
- fixed latest changes
- added news view with provider

### CODE REVIEW

**Comment by BerkSartik** [comment]
Elinize sağlık hocam

File: lib/core/dependency/project_dependency.dart [hunk 1]
```diff
@@ -19,6 +21,10 @@ final class ProjectDependency {
 
 final class ProjectDependencyItems {
```
Comment (burakJs) [inline lib/core/dependency/project_dependency.dart:23]: `@immutable`

File: lib/features/v2/news_jobs/provider/news_jobs_state.dart [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import 'package:equatable/equatable.dart';
+
+final class NewsJobsState extends Equatable {
```
Comment (burakJs) [inline lib/features/v2/news_jobs/provider/news_jobs_state.dart:5]: `@immutable`

File: lib/features/v2/news_jobs/view/sub_view/tab_jobs_view.dart [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:vbaseproject/features/v2/news_jobs/provider/news_jobs_provider.dart';
+import 'package:vbaseproject/product/widget/card/index.dart';
+import 'package:vbaseproject/product/widget/general/index.dart';
+
+final class TabJobsView extends ConsumerWidget {
```
Comment (burakJs) [inline lib/features/v2/news_jobs/view/sub_view/tab_jobs_view.dart:8]: `@immutable`

File: lib/features/v2/news_jobs/view/sub_view/tab_news_view.dart [hunk 4]
```diff
@@ -0,0 +1,26 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:vbaseproject/features/v2/news_jobs/provider/news_jobs_provider.dart';
+import 'package:vbaseproject/product/widget/card/index.dart';
+import 'package:vbaseproject/product/widget/general/general_firestore_list_view.dart';
+
+final class TabNewsView extends ConsumerWidget {
```
Comment (burakJs) [inline lib/features/v2/news_jobs/view/sub_view/tab_news_view.dart:8]: `@immutable`

File: lib/features/v2/news_jobs/view/widget/news_jobs_tab_bar.dart [hunk 5]
```diff
@@ -0,0 +1,15 @@
+part of '../news_jobs_view.dart';
+
+final class _NewsJobsTabBar extends StatelessWidget {
```
Comment (burakJs) [inline lib/features/v2/news_jobs/view/widget/news_jobs_tab_bar.dart:4]: `@immutable`

File: lib/features/v2/news_jobs/view/widget/news_jobs_tab_title.dart [hunk 6]
```diff
@@ -0,0 +1,19 @@
+part of '../news_jobs_view.dart';
+
+final class _NewsJobsTabTitleView extends StatelessWidget {
```
Comment (burakJs) [inline lib/features/v2/news_jobs/view/widget/news_jobs_tab_title.dart:4]: `@immutable`

File: lib/features/v2/news_jobs/view/widget/news_jobs_tab_view.dart [hunk 7]
```diff
@@ -0,0 +1,12 @@
+part of '../news_jobs_view.dart';
+
+final class _NewsJobsTabView extends StatelessWidget {
```
Comment (burakJs) [inline lib/features/v2/news_jobs/view/widget/news_jobs_tab_view.dart:4]: `@immutable`

File: lib/product/widget/card/news_card.dart [hunk 8]
```diff
@@ -5,6 +5,8 @@ import 'package:vbaseproject/product/common/color_common.dart';
 import 'package:vbaseproject/product/package/custom_network_image.dart';
 import 'package:vbaseproject/product/utility/decorations/custom_radius.dart';
 import 'package:vbaseproject/product/utility/padding/page_padding.dart';
+import 'package:vbaseproject/product/widget/general/title/general_big_bold_max_title.dart';
+import 'package:vbaseproject/product/widget/special/user_special_card.dart';
 
 final class NewsCard extends StatelessWidget {
```
Comment (burakJs) [inline lib/product/widget/card/news_card.dart:11]: `@immutable`

File: lib/product/widget/dialog/general_icon_text_dialog.dart [hunk 9]
```diff
@@ -1,6 +1,6 @@
 import 'package:flutter/material.dart';
 import 'package:kartal/kartal.dart';
-import 'package:vbaseproject/product/widget/general/general_sub_title.dart';
+import 'package:vbaseproject/product/widget/general/title/general_sub_title.dart';
 
 /// Dialog contains only text and icon. If you want to show it on the screen, you can use the [show] command.
 final class GeneralIconTextDialog extends StatelessWidget {
```
Comment (burakJs) [inline lib/product/widget/dialog/general_icon_text_dialog.dart:7]: `@immutable`

File: lib/product/widget/dialog/general_text_dialog.dart [hunk 10]
```diff
@@ -1,6 +1,6 @@
 import 'package:flutter/material.dart';
 import 'package:kartal/kartal.dart';
-import 'package:vbaseproject/product/widget/general/general_sub_title.dart';
+import 'package:vbaseproject/product/widget/general/title/general_sub_title.dart';
 
 /// Dialog contains only text. If you want to show it on the screen, you can use the [show] command.
 final class GeneralTextDialog extends StatelessWidget {
```
Comment (burakJs) [inline lib/product/widget/dialog/general_text_dialog.dart:7]: `@immutable`

File: lib/product/widget/general/general_firestore_list_view.dart [hunk 11]
```diff
@@ -0,0 +1,43 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:easy_localization/easy_localization.dart';
+import 'package:firebase_ui_firestore/firebase_ui_firestore.dart';
+import 'package:flutter/material.dart';
+import 'package:vbaseproject/product/init/language/locale_keys.g.dart';
+import 'package:vbaseproject/product/package/shimmer/general_place_shimmer.dart';
+import 'package:vbaseproject/product/widget/lottie/not_found_lottie.dart';
+
+final class GeneralFirestoreListView<T> extends StatelessWidget {
```
Comment (burakJs) [inline lib/product/widget/general/general_firestore_list_view.dart:10]: `@immutable`

File: lib/product/widget/general/title/general_big_bold_max_title.dart [hunk 12]
```diff
@@ -0,0 +1,27 @@
+import 'package:flutter/material.dart';
+import 'package:kartal/kartal.dart';
+
+/// This is a general big title widget with headlineMedium style.
+///
+/// Max line is 2.
+/// Bold and max line.
+final class GeneralBigBoldMaxTitle extends StatelessWidget {
```
Comment (burakJs) [inline lib/product/widget/general/title/general_big_bold_max_title.dart:9]: `@immutable`

File: lib/product/widget/special/user_special_card.dart [hunk 13]
```diff
@@ -0,0 +1,43 @@
+import 'package:flutter/material.dart';
+import 'package:vbaseproject/product/utility/decorations/custom_circle_radius.dart';
+import 'package:vbaseproject/product/utility/padding/page_padding.dart';
+
+final class SpecialUser {
```
Comment (burakJs) [inline lib/product/widget/special/user_special_card.dart:5]: `@immutable`

File: lib/product/widget/special/user_special_card.dart [hunk 14]
```diff
@@ -0,0 +1,43 @@
...
+
+  /// Todo: Add your special user to database
+  static const SpecialUser creator = SpecialUser(
+    name: '@grafikherif',
+    photoUrl:
+        'https://pbs.twimg.com/profile_images/1643695427840098306/ugEFQ49l_400x400.jpg',
+  );
+}
+
+final class UserSpecialCard extends StatelessWidget {
```
Comment (burakJs) [inline lib/product/widget/special/user_special_card.dart:22]: `@immutable`

File: lib/product/widget/text/clickable_title_text.dart [hunk 15]
```diff
@@ -1,6 +1,6 @@
 import 'package:flutter/material.dart';
 import 'package:vbaseproject/product/utility/constants/icon_constants.dart';
-import 'package:vbaseproject/product/widget/general/general_sub_title.dart';
+import 'package:vbaseproject/product/widget/general/title/general_sub_title.dart';
 
 final class ClickableSubTitleText extends StatelessWidget {
```
Comment (burakJs) [inline lib/product/widget/text/clickable_title_text.dart:5]: `@immutable`

**Pull request merged**